res/xml/nepali_phonetic_npl.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<keyboard name="Nepali phonetic" script="nepali">
```

Comment:
The Napali language uses the Devanagari script ? This should be `script="devanagari"`, the filename should be `deva_phonetic_npl.xml`

Reply:
I meant to change the `script` field, not the `name` field.